src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,43 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const clickHandler = (tab) => {
+    if (tab.id !== selectedTabId) {
+      onTabSelected(tab);
+    }
+  };
+
+  const currentTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
```

Comment:
you duplicating this code.
let's set to the state the whole tab not only tab.id so you won't need to use find at all